client/my-sites/checkout/checkout-thank-you/licensing-thank-you-auto-activation.tsx

```diff
@@ -146,6 +158,24 @@ const LicensingActivationThankYou: FC< Props > = ( {
 					)
 				);
 			}
+
+			if ( destinationSiteId === 0 ) {
```

Comment:
`destinationSiteId === 0` means that something went wrong and the product transferring/activation failed.